Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -309,10 +309,18 @@ static void HandleQuery()
 {
   DEBUG_LOG_FMT(GDB_STUB, "gdb: query '{}'", CommandBufferAsString() + 1);
 
-  if (!strcmp((const char*)(s_cmd_bfr + 1), "TStatus"))
-  {
-    return SendReply("T0");
-  }
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "Attached"))
+    return SendReply("1");
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "C"))
+    return SendReply("QC1");
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "fThreadInfo"))
+    return SendReply("m1");
+  else if (!strcmp((const char*)(s_cmd_bfr + 1), "sThreadInfo"))
+    return SendReply("l");
+  else if (!strncmp((const char*)(s_cmd_bfr + 1), "ThreadExtraInfo", 15))
+    return SendReply("00");
+  else if (!strncmp((const char*)(s_cmd_bfr + 1), "Supported", static_cast<size_t>(9)))
+    return SendReply("swbreak+;hwbreak+");
```

Comment:
The implementation looks correct per [the documentation](https://sourceware.org/gdb/onlinedocs/gdb/General-Query-Packets.html#index-qSupported-packet), but I don't think the `static_cast<size_t>(9)` part makes sense; using just `9` is fine.

Reply:
Should I do an strlen instead?

Realized it looked a little ugly (the size_t cast was to fix some warnings I THINK)

Reply:
I see other places using it with just an int, presumably without warnings.  I don't think strlen is necessary (but it [can be inlined at compile time](https://stackoverflow.com/a/26021702), so there isn't a performance cost, though the string would have to be duplicated?)